actions/version_number/action.yaml

```diff
@@ -58,6 +63,8 @@ runs:
           echo "new_version=${{ inputs.version_number }}" >> $GITHUB_OUTPUT
         elif [[ "${{ inputs.type }}" == "Snapshot" ]]; then
           echo "new_version=$NBGV_SimpleVersion-SNAPSHOT" >> $GITHUB_OUTPUT
+        elif [[ "${{ inputs.branch_name }}" == "" || "${{ inputs.branch_name }}" == "master" || "${{ inputs.branch_name }}" == "main" ]]; then
```

Comment:
Isn't this the wrong way round? We don't want the name to include the branch name if it is master or main